README.md

```diff
@@ -6,6 +6,25 @@ If you are looking for the ArcGIS Maps SDK for JavaScript core package, it is av
 
 This repository is organized as a monorepo using yarn workspaces.
 
+## Available Templates
+There are download links in the readme for each of the following templates. We recommend downloading a template from the links below and using your package manager of choice.  
+
```

Comment:
We can probably add to this by copying some of what already exists in https://github.com/Esri/jsapi-resources?tab=readme-ov-file#jsapi-resources